Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	rm -rf bin/*
 
 fmt:
-	gofmt -w $(GOFMT_FILES)
+	goimports -w -local github.com/oracle/terraform-provider-baremetal $(GOFMT_FILES)
```

Comment:
@rcohenma can you include this change in your current rename branch then run `make fmt`.